core/store/models/job_spec_v2.go

```diff
@@ -47,9 +47,9 @@ type (
 		P2PPeerID                              PeerID         `json:"p2pPeerID" toml:"p2pPeerID"         gorm:"column:p2p_peer_id"`
 		P2PBootstrapPeers                      pq.StringArray `json:"p2pBootstrapPeers" toml:"p2pBootstrapPeers" gorm:"column:p2p_bootstrap_peers;type:text[]"`
 		IsBootstrapPeer                        bool           `json:"isBootstrapPeer" toml:"isBootstrapPeer"`
-		EncryptedOCRKeyBundleID                Sha256Hash     `json:"keyBundleID" toml:"keyBundleID"                 gorm:"type:bytea"`
+		EncryptedOCRKeyBundleID                *Sha256Hash    `json:"keyBundleID" toml:"keyBundleID"                 gorm:"type:bytea"`
```

Comment:
have to be pointers because of https://github.com/go-gorm/gorm/issues/2748

Reply:
Can we add a todo comment or add it to the card for removing gormv1?